README.md

```diff
@@ -15,3 +15,8 @@ the function and run automatically when you commit through a GitHub action.
 ## Runtime Analysis
 
 What is the worst-case big $\Theta$ time complexity of your algorithm?
+The worst case  $\Theta$ time complexity of this implementation is  $\Theta(V! * V)$ where $V$ is the number of verticies in the graph. The worst case would become reality in the even that "tryMapping" itterates through all possible mappings(permutations) before finding a valid mapping between the graphs, the number of possible permutations for a graph containing $V$ verticies is $V!$, so this results in $\Theta(V!)$. In a dense graph, the number of neighbors(degree) checked by "isPartialMappingValid" for each permutation approaches the total number of verticies in the graph, resulting in the "*V" term, while this is unlikley, it is the worst case scenario. The helper function "getDegreeCount" itterages over all vertices in the passed graph, and has a worst case complexity of $\Theta(V)$($V =$ the number of vertices). "compareFrequencyMaps" itterates over the degree sequences in the passed maps and has a complexity of $\Theta(n)$ where $n$ is the number of degrees in the sequence. These helper functions are worth noting, but they are ultimatley overshadowed by the "tryMapping" function, so this algorithms worst case $\Theta$ complexity is $\Theta(V! * V)$  
+
+Help: ChatGPT helped with testing and GitHub functionality, I referenced 22dcolli's repository to see if my algorithm was way too inefficient. I also refered back to my isomorphism-nodes-connectivity repo to ensure I was checking all the right criteria.
```

Comment:
I used 22dcolli's repository to basically compare the runtime of my implementation to that of theirs just to see if I was on the right track. The logic behind my code came from my isomorphism-nodes-connectivity to ensure that all the correct conditions were checked, and ChatGPT helped reformat my code to work better with tests, because I had originally implemented a Graph constructor which wasn't wanting to be exported to the test code. ChatGPT also helped implement GitHub actions, and made sure that I was testing all cases in which a graph is isomorphic or not. So in conclusion, I wrote the code in code.js, and ChatGPT helped get rid of the graph constructor, I wrote the test code using the isomorphism-nodes-connectivity repo as a reference, and ran it by ChatGPT for quality assurance, and I used 22dcolli's repository as a reference for peace of mind.